pom.xml

```diff
@@ -935,7 +935,7 @@
     <jetty.maven.version>9.4.34.v20201102</jetty.maven.version>
 
     <!-- security libraries -->
-    <ssp.version>1.2.0</ssp.version>
+    <ssp.version>1.3.0-SNAPSHOT</ssp.version>
```

Comment:
Reminder: I am waiting for the SSP release and will update this version afterwards.

Reply:
I've set this.